README.md

```diff
@@ -195,6 +196,60 @@ buildctl build \
 
 `--local` exposes local source files from client to the builder. `context` and `dockerfile` are the names Dockerfile frontend looks for build context and Dockerfile location.
 
+##### Hint: `dockerfile.v0` frontend with `buildctl build --opt` option
+For `dockerfile.v0` frontend there are some parameters that can be used with `--opt`.
+The following parameters are the most frequently used.
+* target: set the target build stage to build.
+  * Example: `target=Stage1`
+* filename: name of the Dockerfile (Default is 'Dockerfile'),set to use a differently-named Dockerfile.
+  * Example: `filename=dockerfile.prod`,`filename=other.dockerfile`
+* dockerfilekey: name of the custom named context for a directory includes Dockerfile. Useful when there is more than one build source.
+  * Example: `dockerfilekey=dockerfile2`
+  * Full command example: 
+    * `buildctl --frontend=dockerfile.v0 --local dockerfile=. --local dockerfile2=/tmp --opt dockerfilekey=dockerfile2` buildctl will try to use the `/tmp/Dockerfile` as Dockerfile.
```

Comment:
is missing the `build` part in the command. Also it's a little bad formated, because after your command there is more text. It's much nicer in your second full command example starting at line 218

Reply:
tks,good point,I fixed it.